sdk/src/Core/Amazon.Util/Internal/_netstandard/ValueStringBuilder.cs

```diff
@@ -0,0 +1,337 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file is partially adopted from the ValueStringBuilder in the .NET runtime repository. What attribution would you like to see or require?